### DISCUSSION TITLE
review requested pull request filter not working properly

### DISCUSSION THREAD

**Discussion by martinsbalodis** [community/community#59540] [Pull Requests]
### Select Topic Area

Bug

### Body

Just now review requested search filter started glitching. It is showing pull requests that have been recently merged or reviewed. A merged pull request should disappear from this search. 

Steps to reproduce:
1. visit https://github.com/pulls/review-requested
2. merge a pr or review a pr
3. the pr doesn't disappear from https://github.com/pulls/review-requested

![Screenshot from 2023-06-28 12-45-11](https://github.com/community/community/assets/890132/e1a0a602-5767-44d3-af40-99e5fc156425)

**Comment by stianjensen** [comment]
I think this was probably a temporary issue. Haven't seen this pop up recently at least.
